### ISSUE TITLE
Mark stable release with v1.0

### ISSUE THREAD

**Issue by FractalizeR** [gorhill/PHP-FineDiff#28]
It would be good to mark some release with v1.0 tag to use it in composer. Since master can be unstable, it would be good to base our solutions off yours by tag, which will not be changed.

**Comment by gvoss** [comment]
I agree - is there a particular commit we can treat as stable at this time?
